tests/jobs/test_scheduler_job.py

```diff
@@ -1407,7 +1414,9 @@ def test_find_executable_task_instances_task_concurrency(self):  # pylint: disab
         session = settings.Session()
 
         scheduler.dagbag.bag_dag(dag, root_dag=dag)
-        scheduler.dagbag.sync_to_db(session=session)
+        # Since we don't want to store the code for the DAG defined in this file
+        with mock.patch.object(settings, "STORE_DAG_CODE", False):
```

Comment:
Do we have this on every case? If so we should do it in setup with a second patcher (or even if most we should do it in setup)

Reply:
Done in 74850b499